src/main/java/seedu/address/logic/commands/RandomTeamAllocationCommand.java

```diff
@@ -0,0 +1,120 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULECODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NUM_OF_TEAMS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TUTORIALCLASS;
+
+import java.util.ArrayList;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.module.TutorialClass;
+import seedu.address.model.person.Person;
+
+/**
+ * A command to random allocate all students of a particular tutorial class into different teams.
+ */
+public class RandomTeamAllocationCommand extends Command {
+
+    public static final String COMMAND_WORD = "/random_teams";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Randomly allocate all the students in the "
+            + "tutorial class into teams \n"
+            + "Parameters: "
+            + PREFIX_MODULECODE + "MODULE CODE "
+            + PREFIX_TUTORIALCLASS + "TUTORIAL CLASS "
+            + PREFIX_NUM_OF_TEAMS + "NUMBER OF TEAMS "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_MODULECODE + "CS2103T "
+            + PREFIX_TUTORIALCLASS + "T09 "
+            + PREFIX_NUM_OF_TEAMS + "3 ";
+
+    public static final String MESSAGE_MODULE_TUTORIAL_NOT_EXIST = "Please check if you have entered"
+            + " an existing module and tutorial";
+    public static final String MESSAGE_ALLOCATION_NOT_POSSIBLE = "Allocation not possible for tutorial class size %s"
+            + " into %s different teams";
+
+    public static final String MESSAGE_SUCCESS = "Successfully randomly allocate students for tutorial class %s";
+
+    public static final String MESSAGE_SIZE_NEGATIVE = "Number of teams must be more than 0!";
```

Comment:
Perhaps the variable MESSAGE_SIZE_NEGATIVE could be better written as MESSAGE_NO_OF_TEAMS_NEGATIVE?
Since size makes me think about the team size rather than the number of teams

Reply:
Additionally, I think it might be better to refactor these messages into PeopleMessages or TutorialClassMessages since they are quite generic and other functions may benefit from them

Reply:
good catch, edited as according. As for refactoring, I will make a separate PR for that!